gentoo_update/scripts/updater.sh

```diff
@@ -95,11 +95,16 @@ function clean_up() {
 		echo "Cleaning packages that are not part of the tree..."
 		emerge --depclean
 
-		echo "Checking reverse dependencies..."
-		revdep-rebuild
+		if command -v revdep-rebuild >/dev/null 2>&1; then
+			echo "Checking reverse dependencies..."
```

Comment:
Nit, python code is 4 spaces, so you might want to reformat bash to follow the same spacing in the project

Reply:
I am using [shfmt](https://github.com/mvdan/sh) to format Bash, I think there is an `--indent` option that I can use for fix the indentation. Will do it in next release.